server/index.js

```diff
@@ -13,11 +13,22 @@ app.use(expressJson);
 app.use(staticMiddleware);
 app.use(errorMiddleware);
 
+// CORS middleware
+const allowCrossDomain = function (req, res, next) {
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE');
+  res.header('Access-Control-Allow-Headers', 'Content-Type');
+
+  next();
+};
+
+app.use(allowCrossDomain);
```

Comment:
This shouldn't be necessary. If you feel like you really need it, send me a message because I can help you avoid needing it.